### DISCUSSION TITLE
Multiplayer spectating gets stuck loading

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#21549] [Q&A] [answered]
Tried to spectate but even after 2-3 minutes, it was stuck loading.

![osu_2022-12-06_23-01-30](https://user-images.githubusercontent.com/74088315/206092759-1ada0683-2cb7-4c75-8b8a-3d762b886bd1.png)

[database.log](https://github.com/ppy/osu/files/10172941/database.log)
[network.log](https://github.com/ppy/osu/files/10172942/network.log)
[performance.log](https://github.com/ppy/osu/files/10172943/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10172944/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10172945/updater.log)

**Comment by smoogipoo** [comment, marked as answer]
Related: https://github.com/ppy/osu/discussions/21548

A fix has been deployed, but will only affect newly-created rooms.
